ergm.py

```diff
@@ -136,72 +142,166 @@ def approximate_auto_correlation_function(self, networks_sample: np.ndarray) ->
             ).sum(axis=0)
         return auto_correlation_func
 
-    def covariance_matrix_estimation(self, features_of_net_samples: np.ndarray, method='batch', num_batches=25) -> np.ndarray:
+    @staticmethod
+    # @njit
+    def calc_capital_gammas(auto_corr_funcs: np.ndarray) -> np.ndarray:
+        """
+        This is the capital gammas hat from Geyer's handbook of mcmc (1D) and Dai and Jones 2017 (multi-D).
+        They are simply summations over pairs of consecutive even and odd indices of the auto correlation function (gammas).
+        """
+        # From Dai and Jones 2017 - a mean of gamma with its transpose (which corresponds to the negative index with the
+        # same abs value).
+        gamma_tilde = (auto_corr_funcs + np.transpose(auto_corr_funcs, [0, 2, 1])) / 2
```

Comment:
Not sure I understood the gamma hat calculation in the 2017 paper - 

I understand that \Gamma_i =  \gamma_{2i} + \gamma_{2i+1}
but I don't understand the notation of gamma_{t} + gamma_{-t}

however it seems that what you're doing is consistent with the paper, so this is just a general question :)